src/taxonomy/taxonomy-card/TaxonomyCard.scss

```diff
@@ -23,7 +23,7 @@
     -webkit-line-clamp: 6;
   }
 
-  .pgn__card-header-title-md {
+  .taxonomy-card-title {
```

Comment:
This is equivalent to `text-truncate`

Reply:
Thanks! Updated https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/dab9d8a01a0389826e989252b112c3b70869e251